openxr/src/space.rs

```diff
@@ -130,6 +130,109 @@ impl Space {
         ))
     }
 
+    /// Determine the locations of the joints of a hand tracker relative to this space at a
+    /// specified time, if currently known by the runtime.
+    ///
+    /// XR_EXT_hand_tracking must be enabled.
+    #[inline]
+    pub fn locate_hand_joints(
+        &self,
+        tracker: &HandTracker,
+        time: Time,
+    ) -> Result<Vec<HandJointLocation>> {
+        // This assert allows this function to be safe.
+        assert_eq!(&*self.session as *const session::SessionInner, &*tracker.session as *const session::SessionInner,
+                   "`self` and `tracker` must have been created, allocated, or retrieved from the same `Session`");
+        unsafe {
+            let locate_info = sys::HandJointsLocateInfoEXT {
+                ty: sys::HandJointsLocateInfoEXT::TYPE,
+                next: ptr::null(),
+                base_space: self.as_raw(),
+                time,
+            };
+            let mut locations = vec![
```

Comment:
Can we just use a fixed size array here and instead return a `JointLocations` type that contains the array and `is_valid`?